src/js/components/Banner/ShabadavaliBanner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Banner from './Banner';
+
+const ShabadavaliBanner = () => {
+
+  const {
+    isFetchingData: isFetchingBannerMessage,
+    data: bannerData,
+  } = {
+    isFetchingData: false,
+    data: {
+      active: true,
+      data: {
+        message: "Help us preserve the Punjabi language! Donate on Dasvandh Network and double your impact with matching!",
+        link: "dvnetwork.org/learnpunjabi",
+        label: "Donate now",
+        type: "3",
+      }
+    }
+  };
+
+  const lastSeen = sessionStorage.getItem("bannerMessage");
+  const updateLastSeen = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.currentTarget.parentElement?.remove();
+    sessionStorage.setItem("bannerMessage", "seen");
+  };
+
+  if (isFetchingBannerMessage || !bannerData.active || !bannerData.data || lastSeen === 'seen') {
```

Comment:
@MySmilingTurban why we needs to put the `isFetchingBannerMessage` in the condition here? As far as I can see we already have the banner data and not fetching it from network or am I missing something?